src/ProgressBar/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const ProgressBar = ({ totalWidth = "100%", percentComplete = 0 }) => {
```

Comment:
@chris-griffin Thanks for taking a look - this was a good catch!

I kind of broke my own rules here, because the best practice would be to make components fill the width of their parent container, unless otherwise specified by props. We can default this to `100%` and allow consumers to pass in a complete CSS value with the unit